modules/programs/gnome-terminal.nix

```diff
@@ -165,5 +170,8 @@ in
         // mapAttrs' (n: v:
           nameValuePair ("${dconfPath}/profiles:/:${n}") (buildProfileSet v)
         ) cfg.profile;
+
+    programs.bash.initExtra = vteInitStr;
```

Comment:
I had a look at the `vte.sh` code and it seems to clobber `PROMPT_COMMAND` which would break, e.g., direnv and autojump. This should therefore be placed as early as possible in the `initExtra` variable.

So I would suggest changing this to
```nix
programs.bash.initExtra = mkBefore vteInitStr;
```

Reply:
Thanks, didn’t know about `mkBefore`. Fixed.